inspire/testsuite/test_hep.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import make_test_suite, run_test_suite, InvenioTestCase
+from inspire.dojson import processors
+from inspire.dojson.hep import hep2marc
+
+
+class HepRecordsTests(InvenioTestCase):
+    def setUp(self):
+        self.marcxml_to_json = list(processors.convert_marcxml(
+            open('inspire/testsuite/fixtures/test_hep_record.xml',
```

Comment:
Generally, it is safer to use the pkg_resources (like https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_record.py#L40-L42):

```python
marcxml = pkg_resources.resource_string('inspire.testsuite', os.path.join('fixtures', 'test_hep_record.xml'))
```